src/site/content/en/blog/usb/index.md

```diff
@@ -283,6 +283,28 @@ provides a ground-up example of building a USB-controlled LED device designed
 for the WebUSB API (not using an Arduino here). You'll find hardware, software,
 and firmware.
 
+### Revoke access to a USB device {: #revoke-access }
+
+The website can clean up permissions to access a USB device it is no longer
+interested in retaining by calling `forget()` on the `USBDevice` instance. For
```

Comment:
```suggestion
needs by calling `forget()` on the `USBDevice` instance. For
```

Reply:
Fixed in https://github.com/GoogleChrome/web.dev/pull/7401/commits/6a4d237e5a64004abcf5c6dc52e528a30101b5a0